app/controllers/onboardings_controller.rb

```diff
@@ -14,6 +15,11 @@ def show
     set_surrogate_key_header "onboarding-slideshow"
   end
 
+  def suggestions
```

Comment:
Naming is hard. I don't love the name `suggestions` here — we also suggest tags during onboarding, you can follow tags, we're only suggesting users-and-orgs here —  but it's hard to name this accurately and succinctly in our system.

Reply:
:+1: Went with `users_and_organizations` in acb78974c